packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -436,8 +443,8 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
               label={label}
               help={help}
               error={error}
-              onFocus={() => setIsOpened(true)}
-              readOnly
+              onFocus={!readOnly ? () => setIsOpened(true) : undefined}
+              readOnly={readOnly}
```

Comment:
As we discussed IRL, on mobile, this is **ALWAYS** `readOnly`, because it is not really an input, but a trigger to the modal that has the actual input.